uvicorn/supervisors/basereload.py

```diff
@@ -45,6 +45,8 @@ def run(self):
 
     def startup(self):
         message = "Started reloader process [{}]".format(str(self.pid))
+        if self.__class__.__name__ == "WatchGodReload":
+            message += " powered by watchgod"
```

Comment:
```suggestion
            message += " using watchgod"
```

Reply:
Not a fan of peeking into child class type. How about message template being class attribute? It could default to `"Started reloader process [{}]"` but `WatchGodReload` could define custom.